operator/bundle_helpers/README.md

```diff
@@ -2,29 +2,29 @@
 
 For hermetic builds with Konflux, we need to provide the full list of resolved dependencies in `requirements.txt`.
 The dependency source files will be prefetched with Cachi2 and made available to the container image build.
-Because GHA/upstream is running on a different Python version, it uses `requirements-upstream.txt` to mange its own dependencies.
-Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-upstream.txt`.
+Because GHA/upstream is running on a different Python version, it uses `requirements-gha.txt` to mange its own dependencies.
+Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-gha.txt`.
 
 ## Prepare the fully resolved requirements files for Cachi2
 
 ### Prerequisite
 
-Install [pip-compile](https://pip-tools.readthedocs.io/en/stable/).
-
-Example:
+Run the steps inside a container of the same image as the operator-bundle builder stage.
```

Comment:
Here I'd suggest to hint readers at `konflux.bundle.Dockerfile`, that's where they can find the image of the operator-bundle builder stage.